witness.md

```diff
@@ -196,18 +196,19 @@ Next, recursively define the encoding for an Ethereum tree nodes, with some node
 <Leaf_Node(d<65,s<2)> := accountleaf:<Account_Node(d)>^s==0 storageleaf:<Storage_Leaf_Node(d)>^s==1
                          {leaf node accountleaf or storageleaf, depending on whether s==0 or s==1}
 
-<Account_Node(d<65)> := 0x00 pathnibbles:<Nibbles(64-d)> address:<Address> balance:<Bytes32> nonce:<Bytes32>
-                        {account node for externally owned account with values (pathnibbles, address, balance, nonce)}
-                      | 0x01 pathnibbles:<Nibbles(64-d)> address:<Address> balance:<Bytes32> nonce:<Bytes32> code:<Bytecode> storage:<Account_Storage_Tree_Node(0)>
-                        {account node for executed contract account with values (pathnibbles address balance nonce code storage)}
-                      | 0x02 pathnibbles:<Nibbles(64-d)> address:<Address> balance:<Bytes32> nonce:<Bytes32> codehash:<Bytes32> codesize:<U32> storage:<Account_Storage_Tree_Node(0)>
```

Comment:
I think this PR removed the change from #29.

Reply:
Ah I see it has been subsumed into `Bytecode`. I think `Bytecode` will inflate quite a bit with different merklization options.

Reply:
Yes, good point.

This change was in preparation for code merkleization -- everything related to bytecode was put in the rule `<Bytecode>`. This allows the code merkleization spec to define `<Bytecode>` outside of this spec, and make it a drop-in replacement for the current `<Bytecode>`. Once this is defined, we can better evaluate optimizations like this one.

But this reason is not critical. We can change it back if you think it was better before.